documentation/config_examples/helm/README.md

```
@@ -85,14 +91,22 @@ mysql:
+
+redis:
+  enabled: false
+  auth:
+    enabled: true
+    password: "redisPassword"
 ```
 
 > Replace all `REPLACE_WITH_*` values before installing. Do not deploy with the example values.
 > The chart will create the required Kubernetes Secrets during installation using the values provided above.
 > For production environments, consider using pre-created Kubernetes Secrets and reference them instead of storing credentials in `values.yaml`.
 
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)